src/types.ts

```diff
@@ -1,44 +1,39 @@
-export type VoiceStartMessage = {
-  type: 'voiceStart';
+export type MeasureStartMessage = {
+  msgType: 'measureStart';
+  width: string;
```

Comment:
I'm not a fan of using empty string `''` to imply a certain behavior. We need a more advanced type here to allow callers to explicitly not specify a width. For example:

```ts
type Width = {
  widthType: 'unspecified';
} | {
  widthType: 'specified';
  value: number;
  units: 'px' | 'cm' | 'in'; // etc
};
```

Reply:
I suggest to leave it as defined by MusicXML and this is a number. I change it to number.